lib/util/git/config.ts

```diff
@@ -0,0 +1,11 @@
+export type GitNoVerifyOption = 'commit' | 'push';
+
+let noVerify: GitNoVerifyOption[] = [];
+
+export function setNoVerify(value: GitNoVerifyOption[]): void {
+  noVerify = value;
```

Comment:
Please advise on how to best throw that error and which error to throw?

Reply:
Something like:

```ts
if (!is.array(value)) throw new Error(`config error: gitNoVerify`);
```

Reply:
Btw I noticed you guys have `"strictNullChecks": false` :confused:, would probably make sense to set it to true to avoid various buggy behaviors

Reply:
Actually, throwing an error here is incorrect since that means you have skipped runtime type checking of dynamic data elsewhere. This function is told that the type of value will be `GitNoVerifyOption[]`. Instead, whoever populates the config object should validate the types and throw if they are incorrect.